diff -ruN a/Makefile.sharedlibrary b/Makefile.sharedlibrary
--- a/Makefile.sharedlibrary
+++ b/Makefile.sharedlibrary
@@ -45,7 +45,7 @@
 endif
 
 # Change to actual path for actual distribution packaging.
-INSTALL_PREFIX = /usr/local
+INSTALL_PREFIX = $(FLATPAK_DEST)
 
 # The 'noline' variant may be more appropriate for some distributions; it
 # doesn't have #line directives in the combined source.
